src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -160,6 +164,44 @@ public List<TestCase> findTestCasesByMeasureId(String measureId) {
     return findMeasureById(measureId).getTestCases();
   }
 
+  public String deleteTestCase(String measureId, String testCaseId, String username) {
+    if (StringUtils.isBlank(testCaseId) || StringUtils.isBlank(measureId)) {
+      log.info("Test case/Measure Id cannot be null");
+      throw new InvalidIdException("Test case cannot be deleted, please contact the helpdesk");
```

Comment:
Recommend providing a reason for the rejected request in the message.

Reply:
This was something that we had discussed in todays scrum where we don't user to know exactly which field is getting null as that would cause security issues

Reply:
Gotcha. I missed that discussion. Was stating "could not be found" discussed?